### PR TITLE
Layout enhancements

### PR DESCRIPTION
**Pull request by axamei**

### CODE REVIEW

**Comment by axamei** [comment]
Closes #201

File: src/elements/GridItem.vue [hunk 1]
```diff
@@ -25,11 +25,19 @@ export default {
...
     /**
      * Sets the size of the column. Prefix with `sm-` or `lg-`. Based on a 12 column grid.
+     * The inclusion of `auto` will set that grid item to have a width based on the width and height of the content.
      */
     columns: {
       type: String,
       default: "",
     },
+    /**
+     * Sets the vertical alignment of the item. `start`, `center`, or `end`. For horizontal alignment, please look at the wrapper component.
```
Comment (sdellis) [inline src/elements/GridItem.vue]: In the example, you use `v-start`, so maybe we should make sure the comments are consistent.

File: src/elements/GridItem.vue [hunk 2]
```diff
@@ -25,11 +25,19 @@ export default {
...
+     */
+    vertical: {
```
Comment (sdellis) [inline src/elements/GridItem.vue]: I have to say that I'm a bit confused by the `horizontal` prop being on Wrapper and the `vertical` being on GridItem. If I wanted to align the GridItem content, I would have to use a Wrapper?

Comment (axamei) [reply src/elements/GridItem.vue]: There is no `justify-items` or `justify-self` property available on the main axis. The only property available for horizontal alignment in flexbox is `justify-content` which has to be applied to the parent/container of the flexbox item.

This setup is confusing and I'm wondering if we should extract the `flexContainer` out of the `Wrapper` element and create a `GridContainer` element.

File: src/elements/GridItem.vue (hunk 1 above)
Comment (axamei) [reply src/elements/GridItem.vue]: I updated this to be consistent.

File: src/elements/GridItem.vue (hunk 2 above)
Comment (sdellis) [reply src/elements/GridItem.vue]: > This setup is confusing and I'm wondering if we should extract the flexContainer prop out of the Wrapper element and create a GridContainer element.

I'm a fan of this approach. I don't want to add extra work, but it seems like the simplest approach and semantically couples those two components so folks know they should be used together.

**Pull request merged**